### PR TITLE
Test pypy3

### PR DESCRIPTION
**Pull request by altendky**
WIP for:
- [x] #45 required to work

### CODE REVIEW

File: tests/test_make.py [hunk 1]
```diff
@@ -396,6 +396,10 @@ class B:
     assert_dump_load(schema=schema, loaded=loaded, dumped=dumped)
 
 
+@pytest.mark.skipif(
+    condition=sys.implementation.name == "pypy" and sys.pypy_version_info < (7, 2),
+    reason="Forward references and string annotations are broken.",
+)
```
Comment (altendky) [inline tests/test_make.py]: I'm not totally clear on when it should be `xfail` vs. `skipif`.

**Comment by altendky** [comment]
Hmm...  I should try pulling this out and see what we get from Travis.

https://github.com/python-desert/desert/blob/0570e0f50131ec8cbdc218697543e0bb049072d6/.travis.yml#L38-L49

**Comment by altendky** [comment]
Try to get some clean reference build times.

**Comment by altendky** [comment]
Maybe I will get annoyed at these times myself, sooner than later.

**Comment by altendky** [comment]
Hmm, I should probably actually dig into the coverage thing.  Though I did see a failed report somewhere so I'll trigger a build in case that's all.

**Comment by altendky** [comment]
Yep, the rebuild seemed to get us back to reasonable numbers.  Slightly improved actually, due to the added test xfail I'd guess?  Though that points out that I ought to look at why it's not 100%...  but that's separate.

**Review by ghost — APPROVED** [review]

**Pull request merged**